app/services/cancel_rdv_by_agent_service.rb

```diff
@@ -0,0 +1,15 @@
+class CancelRdvByAgentService
+  attr_reader :rdv
+
+  def initialize(rdv)
+    @rdv = rdv
+  end
+
+  def perform
+    @rdv.update(status: :excused, cancelled_at: Time.now)
+    @rdv.users.map(&:user_to_notify).uniq.each do |user|
+      RdvMailer.cancellation(@rdv, user).deliver_later if user.email.present?
```

Comment:
the mailer method name would deserve to refer to the agent canceling the RDV for more clarity imho

Reply:
`agent_cancel_rdv` ?